nuxt.config.js

```diff
@@ -14,13 +14,16 @@ module.exports = NuxtHelpers([
       { hid: 'description', name: 'description', content: 'Nuxt.js project' }
     ],
     link: [
-      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
+      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
+      { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css?family=Montserrat:200,400' },
+      { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css?family=Droid+Sans:400,700' },
+      { rel: 'javascript', href: 'https://use.fontawesome.com/393a6fea12.js' }
```

Comment:
This is good for now, but once we upgrade to Nuxt 1.0 and Nuxt modules are stable, we should switch to using the [Font Awesome module](https://github.com/nuxt/modules/tree/master/modules/font-awesome).